sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,712 @@
+        Response response) {
+         // pageCollection enables enumerating list of chat members.
+         pageCollection.getFirstPage(new Callback<Page<ReadReceipt>>() {
+            @Override
+            public void onSuccess(Page<ReadReceipt> firstPage, Response response) {
+                for (ReadReceipt receipt : firstPage.getItems()) {
+                    //TODO: Take further action
+                }
+                retrieveNextReceiptsPages(firstPage.getPageId(), pageCollection);
+            }
+
+            @Override
+            public void onFailure(Throwable throwable, Response response) {
+                // TODO: Display error message.
+            }
+         }
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+
+void retrieveNextReceiptsPages(String nextPageId,
+                              AsyncPagedDataCollection<ReadReceipt, Page<ReadReceipt>> pageCollection) {
+    pageCollection.getPage(nextPageId, new Callback<Page<ReadReceipt>>() {
+        @Override
+        public void onSuccess(Page<ReadReceipt> nextPage, Response response) {
+            for (ReadReceipt receipt : nextPage.getItems()) {
+                //TODO: Take further action
+            }
+            if (nextPage.getPageId() != null) {
+                retrieveNextReceiptsPages(nextPage.getPageId(), pageCollection);
+            }
+        }
+
+        @Override
+        public void onFailure(Throwable throwable, Response response) {
+            // TODO: Display error message.
+        }
+    });
+}
+```
+
+# Troubleshooting
+
+## General
+
+The client raises via Callback.onFailure.
+
+```java
+client.createChatThread(thread, new Callback<MultiStatusResponse>() {
+    public void onFailure(Throwable throwable, okhttp3.Response response) {
+        // TODO: Display error message.
+    }
+});
+```
+
+# Next steps
+
+More sample code should go here, along with links out to the appropriate example tests.
+
+# Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+![Impressions](TODO: Find impressions URL)
```

Comment:
Just copy the URL from another library (e.g. storage) and change the URL-encoded parts of the path to match the path of this README.

Reply:
thanks, updated.